.github/workflows/percy_snapshots.yml

```diff
@@ -0,0 +1,30 @@
+name: Percy Snapshots
+
+on: 
+  push:
+    branches-ignore: dev/*
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [15.x]
```

Comment:
Should we use node 12? I think that's what most people on the team are using.

Reply:
percy snapshots aren't really something we want to take locally, so I don't think there's any need to use an older version?

Reply:
tbh I just I want to use the newest, shiniest, fastest toys possible, and have an ingrained fear of being left behind